### PR TITLE
event_manager: Describe events with dynamic size in doc

### PR DESCRIPTION
**Pull request by MarekPieta**
Change describes events with dynamic data size in documentation.

Jira:DESK-1108

### CODE REVIEW

**Review by pdunaj — APPROVED** [review]

**Comment by greg-fer** [comment]
The review will be handled by @FrancescoSer .

**Review by FrancescoSer — APPROVED** [review]
doc reviewed

File: include/event_manager.rst [hunk 1]
```diff
@@ -191,12 +191,14 @@ The following code example shows a source file for the event type ``sample_event
 Creating a listener
 *******************
 
-Modules that should receive events managed by the Event Manager must be registered as listeners and subscribe to a given event type.
+Any Module that has to receive events managed by the Event Manager must be registered as listener and must be subscribed to a given event type.
```
Comment (MarekPieta) [inline include/event_manager.rst:194]: Can we use "A module ..." here ?

Comment (MarekPieta) [reply include/event_manager.rst]: discussed offline

**Pull request merged**